src/System.Management.Automation/resources/SessionStateStrings.resx

```diff
@@ -279,6 +279,9 @@
   <data name="GetContentWriterDynamicParametersProviderException" xml:space="preserve">
     <value>The dynamic parameters for the GetContentWriter operation cannot be retrieved for the '{0}' provider for path '{1}'. {2}</value>
   </data>
+  <data name="GetContainerContentException" xml:space="preserve">
+    <value>Unable to access '{0}' contents as it is a container. Please use 'Get-ChildItem {0}' instead.</value>
```

Comment:
Thanks for taking this on. Given that this message is specific to the filesystem provider, I suggest using _directory_ instead of _container_ for clarity. Perhaps something like the following?

> '{0}' is a directory, not a file. To list the items contained in a directory, use 'Get-ChildItem'.

Reply:
@mklement0 sounds good, will change it the container part. I believe giving `Get-ChildItem <the input path>` in error message will be more helpful than just giving the cmdlet name to use.

your thoughts ?

Reply:
Thanks. 

I personally think it's cleaner (less noisy) without the path, and is generally better to be concise in error message - without leaving out vital information, obviously, which is always a balancing act.

In this particular case, if the user managed to call `Get-Content someDir` and ran into the error, I think they 'll be able to infer from mentioning only the _name_ of the cmdlet how to apply it _analogously_.

I don't feel strongly about this, however.

@SteveL-MSFT, what do you think?

Reply:
This is provider specific issue and it appears only the FileSystemProvider supports IContentCmdletProvider.  I think the error string should be in FileSystem Provider resx and I'm fine with @mklement0's suggested error message.